error

```
time="2026-04-15T02:59:47Z" level=info msg="Attempting to download the image" arch=x86_64 digest="sha256:6205ae0c524b4d1816dbd3573ce29b5c44ed26c9fbc874fbe48c41c89dd0bac2" location="https://download.fedoraproject.org/pub/fedora/linux/releases/41/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2"
time="2026-04-15T02:59:48Z" level=fatal msg="failed to download \"[https://download.fedoraproject.org/pub/fedora/linux/releases/41/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2\](https://download.fedoraproject.org/pub/fedora/linux/releases/41/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2/)": unexpected HTTP status Not Found, body=\"<!DOCTYPE HTML PUBLIC \\\"-//IETF//DTD HTML 2.0//EN\\\">\\n<html><head>\\n<title>404 Not Found</title>\\n</head><body>\\n<h1>Not Found</h1>\\n<p>The requested URL was not found on this server.</p>\\n<hr>\\n<address>Apache Server at ftp2.osuosl.org Port 443</address>\\n</body></html>\\n\""
[ERROR] Failed to create Lima VM definition. Check logs.
```

Thank you for the PR.

Looking at the error, it seems that the Fedora image is no longer available.
Also, Fedora 41 appears to have reached end of life.

https://docs.fedoraproject.org/en-US/releases/eol/

Because of that, I don't think simply updating the comment would be meaningful.
Would it be possible to upgrade this to a newer version, such as Fedora 43?

If you are currently busy, I can close this PR for now and create a separate issue to track the Fedora 43 upgrade.

--- a/experiment/selinux/lima-setup.sh
+++ b/experiment/selinux/lima-setup.sh
@@ -137,17 +137,14 @@ cat > "$TEMP_DIR/lima-fedora.yaml" << 'LIMA_YAML'
 
 # Using official Fedora Cloud Base Images.
 # Replace with a specific version if needed, check https://getfedora.org/en/cloud/download/
-# Using Fedora 39 as Fedora 40 URL might not be stable yet
+# Using Fedora 43 URLs
 images:
-- location: https://download.fedoraproject.org/pub/fedora/linux/releases/41/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2
+- location: https://download.fedoraproject.org/pub/fedora/linux/releases/43/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-43-1.6.x86_64.qcow2
   arch: x86_64
-  digest: sha256:6205ae0c524b4d1816dbd3573ce29b5c44ed26c9fbc874fbe48c41c89dd0bac2
-- location: https://download.fedoraproject.org/pub/fedora/linux/releases/41/Cloud/aarch64/images/Fedora-Cloud-Base-Generic-41-1.4.aarch64.qcow2
+  digest: sha256:846574c8a97cd2d8dc1f231062d73107cc85cbbbda56335e264a46e3a6c8ab2f
+- location: https://download.fedoraproject.org/pub/fedora/linux/releases/43/Cloud/aarch64/images/Fedora-Cloud-Base-Generic-43-1.6.aarch64.qcow2
   arch: aarch64
-  digest: sha256:085883b42c7e3b980e366a1fe006cd0ff15877f7e6e984426f3c6c67c7cc2faa
-- location: https://dl.fedoraproject.org/pub/alt/risc-v/release/41/Cloud/riscv64/images/Fedora-Cloud-Base-Generic-41.20250224-1026a2d0e311.riscv64.qcow2
-  arch: riscv64
-  digest: sha256:6a8272a858d7f1498f49ce362b34f0b9b959885f63285158947e045abfeece40
+  digest: sha256:66031aea9ec61e6d0d5bba12b9454e80ca94e8a79c913d37ded4c60311705b8b
 
 cpus: __CPUS__
 memory: "__MEMORY__"
@@ -207,7 +204,7 @@ provision:
              source "$HOME/.cargo/env"
         fi
       fi
-      
+
       echo "Ensuring $HOME/.cargo/env is sourced in .bashrc for future logins..."
       if ! grep -q 'source "$HOME/.cargo/env"' "$HOME/.bashrc"; then
         echo 'source "$HOME/.cargo/env"' >> "$HOME/.bashrc"

